### PR TITLE
chore(data): add vedant nandwana as volunteer

### PR DESCRIPTION
**Pull request by vedantnn71**

### CODE REVIEW

**Review by accodes21 — COMMENTED** [review]
Thanks for volunteering Vedant.
Can you pls limit your topics to max 2-3

File: data/users/vedantnn7.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Vedant Nandwana",
+    "bio": "Fullstack Developer",
+    "image": "https://pbs.twimg.com/profile_images/1523167477315690497/mqLWnX_e_400x400.png",
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["typescript", "docker", "postgres", "nextjs"]
```
Comment (codewithvoid) [inline data/users/vedantnn7.json:8]: We don't have categories for `docker` and `postgres` yet Vedant.

You can add them back once we add those.

File: data/users/vedantnn7.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Vedant Nandwana",
+    "bio": "Fullstack Developer",
+    "image": "https://pbs.twimg.com/profile_images/1523167477315690497/mqLWnX_e_400x400.png",
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["web development", "software development", "reactjs", "nodejs", "rust"]
```
Comment (codewithvoid) [inline data/users/vedantnn7.json:8]: can you remove web-development and software development from topics and rust in programming category.

updated here
```json
  "categories": [
    {
      "slug": "web-development",
      "topics": [
        "ReactJS",
        "NodeJS"
      ]
    },
    {
      "slug": "programming",
      "topics": [
        "rust"
      ]
    }
  ]
```

**Comment by codewithvoid** [comment]
Perfect. Thank you!

**Pull request merged**